webapp/components/wpt-results.html

```diff
@@ -126,7 +126,7 @@
           <template is="dom-repeat" items="{{displayedNodes}}" as="node">
             <tr>
               <td>
-                <path-part path="{{ node.path }}" is-dir="{{ node.isDir }}" navigate="{{ bindNavigate() }}"></path-part>
+                <path-part prefix="/results" path="{{ node.path }}" is-dir="{{ node.isDir }}" navigate="{{ bindNavigate() }}"></path-part>
```

Comment:
Documented as "Domain path-prefix, e.g. '/interop/'", so this already existed, then :)

Reply:
Yes, planned in advance.